tools/internal_ci/helper_scripts/prepare_build_macos_rc

```diff
@@ -104,20 +81,8 @@ then
   time pip install --user virtualenv
   time pip install --user --upgrade Mako tox setuptools==44.1.1 twisted pyyaml pyjwt cryptography requests
 
-  # make sure md5sum is available (requires coreutils 8.31+)
-  brew update-reset
-  brew upgrade coreutils
-
-  # Install Python 3.7, 3.8, 3.9
-  time curl -O https://www.python.org/ftp/python/3.7.0/python-3.7.0-macosx10.9.pkg
-  time curl -O https://www.python.org/ftp/python/3.8.0/python-3.8.0-macosx10.9.pkg
+  # Install Python 3.9
   time curl -O https://www.python.org/ftp/python/3.9.0/python-3.9.0-macosx10.9.pkg
-  echo "ae0717a02efea3b0eb34aadc680dc498 python-3.7.0-macosx10.9.pkg" > /tmp/python_installer_checksum.md5
-  echo "f5f9ae9f416170c6355cab7256bb75b5 python-3.8.0-macosx10.9.pkg" >> /tmp/python_installer_checksum.md5
-  echo "16ca86fa3467e75bade26b8a9703c27f python-3.9.0-macosx10.9.pkg" >> /tmp/python_installer_checksum.md5
-  md5sum -c /tmp/python_installer_checksum.md5
```

Comment:
We still want line 117-118 to validate the installer's hash.

Reply:
I have some idea to do it with a different tool. Let me try something.

Reply:
A builtin shasum appears to work so it's replacing md5sum.